SwiftStyleGuide.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  SwiftStyleGuide.swift
+//  PurchasesCoreSwift
+//
+//  Created by Andrés Boedo on 7/12/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+protocol ValuePrintable {
+    // make protocol variables read-only unless needed
+    func foo() -> String
+}
+
+// use protocol extensions for default implementations of protocols for types
+extension ValuePrintable where Self: NSNumber {
+    func foo() -> String {
+        return "the number as float is: \(self.floatValue)"
+    }
+}
+
+// prefer structs to classes whenever possible.
+// keep in mind that structs are value types.
+// More info here: https://developer.apple.com/documentation/swift/choosing_between_structures_and_classes
+struct MyStruct {
+
```

Comment:
Should we enforce a new line after class/struct/protocol/extension definition + new line before end of entity? (Google does this)

Reply:
sure! I like it

Reply:
surprisingly, I can't find a swiftlint rule for this, so I'll add it as a comment